manifests/params.pp

```diff
@@ -33,6 +33,7 @@
   # Webhook configuration information
   $webhook_user               = 'puppet'
   $webhook_pass               = 'puppet'
+  $webhook_bind_address       = $::ipaddress
```

Comment:
This should default to '0.0.0.0'. Otherwise the hook won't bind on all interfaces, as is the expected behavior.